.github/native-tests.json

```diff
@@ -137,6 +137,12 @@
             "timeout": 20,
             "test-modules": "hibernate-validator",
             "os-name": "windows-latest"
+        },
+        {
+            "category": "DevTools Integration Tests",
+            "timeout": 70,
```

Comment:
70 minutes? How long does it take on average?

Reply:
2-3 minutes, maybe I should lower that a bit :)